_layouts/packages.html

```diff
@@ -0,0 +1,92 @@
+---
+layout: start
+---
+
+{% capture manual %}{{site.data.gap.GAPManualLink | append: "/pkg/" | append: page.dirname}}{% endcapture %}
+
+<h2>GAP package {{ page.name }}</h2>
+
+<h2>{{ page.description }}</h2>
+
+<p class="homelink">[<a href="{{ page.www }}">WWW homepage</a>]</p>
+
+{% if page.authors.size > 1 %}
+    <h4>Authors</h4>
+{% else %}
+    <h4>Author</h4>
+{% endif %}
+{% assign persons = "" %}
+{% for person in page.authors %}
+    {% if person.url %}
+        {% capture persons %}{{ persons }}<a href="{{ person.url }}">{{ person.name }}</a>|{% endcapture %}
+    {% else %}
+        {% capture persons %}{{ persons }}{{person.name}}|{% endcapture %}
+    {% endif %}
+{% endfor %}
+<p>{{ persons | split: "|" | join: ", " }}</p>
+
+{% if page.maintainers.size > 1 %}
+    <h4>Maintainer</h4>
+{% else %}
+    <h4>Maintainers</h4>
+{% endif %}
+{% assign persons = "" %}
+{% for person in page.maintainers %}
+    {% if person.url %}
+        {% capture persons %}{{ persons }}<a href="{{ person.url }}">{{ person.name }}</a>|{% endcapture %}
+    {% else %}
+        {% capture persons %}{{ persons }}{{person.name}}|{% endcapture %}
+    {% endif %}
+{% endfor %}
+<p>{{ persons | split: "|" | join: ", " }}</p>
+
+<h4>Short Description</h4>
+<p>{{ page.abstract }}</p>
+
+<h4>Version</h4>
+<p> Current version shipped with GAP: {{ page.version }} &nbsp;&nbsp;(Released  {{ page.date | date: "%d/%m/%Y"}})</p>
+
+<!-- TODO href to license empty-->
+<h4>License</h4>
+<p><a href="{{ page.licenseURL }}">{{ page.license }}</a></p>
```

Comment:
I don't think it's realistic at this point to expect a licenseURL. It would make sense to extend PackageInfo.g to carry the relevant info (e.g. add a `LicenseFile` field which contains the name of the license file of a package). But it's in general not possible to deduce this from the `License` field alone, as that can be any SPDX identifier, which includes multi licensing variants.

Reply:
we drop that and just print the `License` field